apps/utils/factories/team.py

```diff
@@ -19,6 +19,13 @@ class Meta:
     team = factory.SubFactory(TeamFactory)
     role = "admin"
 
+    @factory.post_generation
```

Comment:
These factories are only used in tests right?

Reply:
Thoughts on having an `apps/utils/tests/factories` file and keep the test related utils in there?

Reply:
Could do. I don't think the location matters too much.

Reply:
I usually just like to keep test related things in some sort of test/ folder, but it's fine. Agreed that it doesn't matter too much